Makefile

```diff
@@ -0,0 +1,7 @@
+test:
+	elm-verify-examples && elm-test
+
+format:
+	elm-format --yes src
+
+.PHONY: test format
```

Comment:
```suggestion
```

I prefer having `.PHONY` above the target - it's easier to maintain in my opinion.